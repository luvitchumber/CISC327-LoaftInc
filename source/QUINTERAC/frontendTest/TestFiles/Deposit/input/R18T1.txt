deposit 1234567 10000
