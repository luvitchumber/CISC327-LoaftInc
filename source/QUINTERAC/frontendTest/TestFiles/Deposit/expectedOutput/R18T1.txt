DEP 1234567 10000 0000000 ***